docs/LV_client/tools/mono_radar_plots.md

```diff
@@ -0,0 +1,61 @@
+# mono_radar_plots.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_radar_plots.png"/>
+</p>
+
+### Description
+
+Tool to plot the range, aoa and velocity to compare the outpurs of the mono_radar.vi and the mono_ground_radar.vi for a specific Tracking element on the list of target of the ground truth radar sensor.
```

Comment:
typo: outputs.

Reply:
Some parts of this were capitalized in previous files. Tool plots the range, aoa and velocity in order to compare the outputs of the mono_radar.vi and the mono_ground_radar.vi for a specific tracking element on the list of targets part of the ground truth radar sensor.